base/applications/notepad/dialog.c

```diff
@@ -1072,6 +1072,16 @@ VOID DIALOG_Search(VOID)
 
 VOID DIALOG_SearchNext(VOID)
 {
+    Globals.find.Flags |= FR_DOWN;
+    if (Globals.find.lpstrFindWhat != NULL)
+        NOTEPAD_FindNext(&Globals.find, FALSE, TRUE);
+    else
+        DIALOG_Search();
+}
+
+VOID DIALOG_SearchPrev(VOID)
```

Comment:
Both functions could be merged together, with a BOOL parameter that just specifies the direction of the search.

Reply:
Fixed.